src/main/java/ca/corefacility/bioinformatics/irida/ria/utilities/FileUtilities.java

```diff
@@ -347,7 +347,7 @@ public static ExcelData parseExcelFile(AnalysisOutputFile outputFile, int sheetI
 					if (cell == null) {
 						cell = new StreamingCell(i, row.getRowNum(), false);
 					}
-					CellType cellType = cell.getCellTypeEnum();
+					CellType cellType = cell.getCellType();
```

Comment:
`getCellTypeEnum()` was deprecated and replaced with `getCellType()` - https://poi.apache.org/apidocs/dev/org/apache/poi/ss/usermodel/Cell.html#getCellTypeEnum--